### PR TITLE
fix: server_logs creates inf loop on l-s install

### PR DESCRIPTION
**Pull request by mautematico**

### CODE REVIEW

**Comment by mautematico** [comment]
[Saving server logs to db creates a infinite loop on initial l-s install.](https://trello.com/c/HpmFDkm3/160-saving-server-logs-to-db-creates-a-infinite-loop-on-initial-l-s-install)

**Comment by RafaelTaranto** [comment]
@mautematico we don't want to change migration order.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]
We don't want to change migration order

File: bin/lamassu-migrate [hunk 1]
```diff
@@ -23,10 +23,21 @@ const getMigrateFile = () => {
...
+  })
   .then(([qResult, migrateFile]) => {
     if (!qResult && migrateFile) {
       return db.none('insert into migrations (id, data) values (1, $1)', [migrateFile])
     }
+  })
+  .then(() => {
+
+
+
```
Comment (RafaelTaranto) [inline bin/lamassu-migrate]: Remove this empty promise.

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: bin/lamassu-migrate (hunk 1 above)
Comment (mautematico) [reply bin/lamassu-migrate]: lol, shame on me!

done, empty promise removed, and actually no additional promise was needed so I merged the other two into a single one.

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**